test/functionalTest/cases/3468_v2_registrations_attribute_name_with_forbidden_character/attribute_name_with_forbidden_character.test

```diff
@@ -0,0 +1,81 @@
+# Copyright 2020 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+POST /v2/registrations Attribute name with whitespace
+
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB 0
+
+--SHELL--
+
+#
+# 01. POST /v2/registrations Forbidden character in attribute name
```

Comment:
To make the test even better :) I'd suggest to add some more cases:

```
 # 02. POST /v2/registrations Forbidden character in attribute type
 # 03. POST /v2/registrations Forbidden character in entity id
 # 04. POST /v2/registrations Forbidden character in entity type
```

Reply:
Hi @fgalan 
> To make the test even better :) I'd suggest to add some more cases:
# 02. POST /v2/registrations Forbidden character in attribute type
# 03. POST /v2/registrations Forbidden character in entity id
# 04. POST /v2/registrations Forbidden character in entity type

As per “POST /v2/registrations” API payload mentioned in API walkthrough, I’m able to create test cases for forbidden  character in “entity id” and “entity type”.
But in case of  forbidden character in “attribute type” I’m unable to create it because “attribute type” is not present in “POST /v2/registrations” API  payload.

   "attrs": [
      "temperature",
      "pressure"
    ]

Even I’ve tried with adding “attribute type” in payload but unable to create it.
I’ve checked “POST /v1/registry” API payload, in this payload “attribute type” is present.
Please confirm my understanding.

         "attributes": [
                {
                    "name": "temperature",
                    "type": "float",
                    "isDomain": "false"
                },
                {
                    "name": "pressure",
                    "type": "integer",
                    "isDomain": "false"
                }
            ],

Reply:
Yes, you are right. Please ignore the attributes type case in my original suggestion.

Reply:
Hi @fgalan 
I've added new test case for entity type and entity id